src/d3d12/d3d12_resource_pool_texture.hpp

```diff
@@ -59,7 +59,7 @@ namespace wr
 		DescriptorAllocator* m_mipmapping_allocator;
 
 		//Track resources that are created in one frame and destroyed after
-		std::vector<d3d12::TextureResource*> m_temporary_textures;
+		std::vector<d3d12::TextureResource*> m_temporary_textures[d3d12::settings::num_back_buffers];
```

Comment:
we not programming C we are programming C++. prefer std array

Reply:
done 👍